pkg/chezmoi/archive.go

```diff
@@ -132,10 +134,44 @@ func isTarArchive(r io.Reader) bool {
 	return err == nil
 }
 
+func ImplicitDirHeader(dir string, fileHeader *tar.Header) *tar.Header {
```

Comment:
This function should be called `implicitDirHeader` (lowercase `i`) as it's for package internal use only (the uppercase `I` makes it exported).